eng/scripts/Create-APIView.ps1

```diff
@@ -163,7 +170,7 @@ function New-SwaggerAPIViewTokens {
 
   # Get Changed Swagger Files
   LogInfo " Getting changed swagger files in PR, between $SourceCommitId and $TargetCommitId"
-  $changedFiles = Get-ChangedFiles -baseCommitish $SourceCommitId -targetCommitish $TargetCommitId
+  $changedFiles = Get-ChangedFiles
```

Comment:
Why did you change this now? Are you now only looking at the list of files changes in the PR?

Reply:
Right. I am allowing this to use the default parameters which is using `HEAD^ & HEAD` which is the merge commit and the target branch commit. That seems to be the right way to get files changed in the PR.

However the swagger tool itself is run on the source branch commit and then the target branch